src/fabmap.cpp

```diff
@@ -51,623 +51,534 @@
 // possibility of such damage.
 //////////////////////////////////////////////////////////////////////////////*/
 
-#include "../include/openfabmap.hpp"
-
-using std::vector;
-using std::list;
-using std::map;
-using std::multiset;
-using std::valarray;
-using cv::Mat;
+#include "../include/fabmap.hpp"
 
 /*
-	Calculate the sum of two log likelihoods
+    Calculate the sum of two log likelihoods
 */
-double logsumexp(double a, double b) {
-	return a > b ? log(1 + exp(b - a)) + a : log(1 + exp(a - b)) + b;
+namespace of2 {
```

Comment:
originally I had the logsumexp outside the namespace to not expose the function as it is an internal function. I'm not sure if putting it inside changes things or if it is necessary. I am happy either way if there is a reason.